notebooks/notebook_template_review.py

```diff
@@ -193,71 +194,81 @@ def parse_notebook(path: str) -> None:
         
         cells = content['cells']
         
-        NotebookRule.init()
+        NotebookRule.init(path, content['cells'])
         
-        CopyrightRule().validate(path, cells)
+        CopyrightRule().validate()
```

Comment:
You should instantiate outside this method and just call them here.

Reply:
not sure what the comment means. Can you give an example?